ringpop.go

```diff
@@ -62,11 +61,18 @@ type Interface interface {
 	Forward(dest string, keys []string, request []byte, service, endpoint string, format tchannel.Format, opts *forward.Options) ([]byte, error)
 
 	Labels() (*swim.NodeLabels, error)
+
+	// events.EventRegistar
+	RegisterListener(l events.EventListener)
```

Comment:
when it was embedded the mock generation with our `mockery` version broke because it didn't make the mocked function receivers for these functions. I think I left a comment for it somewhere but shall add it here as well.